packages/metrics/src/__tests__/metrics.spec.ts

```
@@ -0,0 +1,97 @@
+// metrics.test.ts
+import { metrics } from '../metrics.js';
+import client from 'prom-client';
+
+describe('Metrics singleton', () => {
+  beforeEach(() => {
+    // Réinitialise toutes les métriques entre chaque test
+    client.register.resetMetrics();
+    // Reset internal flag so register() se relance
+    // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)